packages/cascara/src/modules/DataEmail/DataEmail.js

```diff
@@ -29,38 +31,35 @@ const DataEmail = ({
 }) => {
   const { isEditing, formMethods } = useContext(ModuleContext);
 
+  // do not add aria label if tag is used
+  const conditionalLabelProps = getConditionalLabelProps(label, isLabeled);
+
   const renderEditing = (
-    <label htmlFor={label}>
-      {label && isLabeled && <span className={styles.LabelText}>{label}</span>}
-      <Input
-        {...rest}
-        aria-label={label}
-        className={styles.Input}
-        defaultValue={value}
-        id={label}
-        name={attribute || label}
-        ref={formMethods?.register}
-        type='email'
-      />
-    </label>
+    <Input
+      {...conditionalLabelProps}
+      {...rest}
+      className={styles.Input}
+      defaultValue={value}
+      id={attribute || label}
+      name={attribute || label}
+      ref={formMethods?.register}
+      type='email'
+    />
   );
 
   const renderDisplay = (
-    <span>
-      {label && isLabeled && <span className={styles.LabelText}>{label}</span>}
-      <span className={styles.Input} {...rest}>
-        {value}
-      </span>
+    <span {...rest} aria-label={label} className={styles.Input}>
+      {value}
     </span>
```

Comment:
@lordpixel what happened to the input on renderDisplay in this module? It is gone.